### PR TITLE
enable debug

### PR DESCRIPTION
**Pull request by adizaimi**

### CODE REVIEW

File: db/sqlinterfaces.c [hunk 1]
```diff
@@ -7805,8 +7805,7 @@ int handle_newsql_requests(struct thr_handle *thr_self, SBUF2 *sb)
         goto done;
 
 #if DEBUG
```
Comment (nirbhayc) [inline db/sqlinterfaces.c]: s/#if/#ifdef ?

Comment (adizaimi) [inline db/sqlinterfaces.c]: will do. Thanks

File: db/db_util.c [hunk 2]
```diff
@@ -442,8 +442,8 @@ int rewrite_lrl_un_llmeta(const char *p_lrl_fname_in,
 
     /* add table definitions */
     for (i = 0; i < num_tables; ++i) {
-        if (strncmp(p_table_names[i], "sqlite_stat", sizeof("sqlite_stat")-1) ==
-            0)
```
Comment (nirbhayc) [inline db/db_util.c]: is_sqlite_stat(p_table_names[i]) ?

File: db/sqlglue.c [hunk 3]
```diff
@@ -7220,7 +7180,7 @@ sqlite3BtreeCursor_analyze(Btree *pBt,      /* The btree */
     struct dbtable *db;
 
     assert(iTable >= RTPAGE_START);
-    assert(iTable < (thd->rootpage_nentries + RTPAGE_START));
+    /* INVALID: assert(iTable < (thd->rootpage_nentries + RTPAGE_START)); */
```
Comment (nirbhayc) [inline db/sqlglue.c]: Why?

File: sqlite/vdbemem.c [hunk 4]
```diff
@@ -71,6 +71,7 @@ int sqlite3VdbeCheckMemInvariants(Mem *p){
       ((p->flags&MEM_Static)!=0 ? 1 : 0) >= 1
     );
   }
+  */
```
Comment (nirbhayc) [inline sqlite/vdbemem.c]: Add a TODO?

**Pull request merged**